src/sentry/integrations/notifydisable.py

```diff
@@ -11,6 +11,9 @@
 
 
 def get_url(organization: Organization, provider_type: str, provider: str) -> str:
+    if not provider_type:
+        return str(organization.absolute_url("/settings/"))
```

Comment:
Oh sorry, this should be `/settings/integrations/` if no provider type, the integration directory is like https://sentry.sentry.io/settings/integrations/ which lists integrations, plugins, and published sentry apps.